part_one.sql

```diff
@@ -1 +1,93 @@
 
+/* Prompt 1 */
+
+SELECT
+   document.id AS document_id 
+FROM
+   document 
+   LEFT JOIN
+      (
+         SELECT DISTINCT
+            document_id 
+         FROM
+            `page`
+      )
+      AS pages 
+      ON pages.document_id = document.id 
+WHERE
+   pages.document_id IS NULL;
```

Comment:
Since there is no need for aggregation (and there isn't much of a performance difference), `SELECT DISTINCT` seems a bit more readable than `GROUP BY`